app/conf/userConf.go

```diff
@@ -6,18 +6,30 @@ import (
 	"os"
 	"regexp"
 	"errors"
+	"time"
 )
 
-// Config the application's configuration
-type Config struct {
-	NavConfPath       	string
-	RunningNavVersion 	string
-	RpcUser 				string
-	RpcPassword 			string
+// UserConfig the application's configuration
+type UserConfig struct {
+	NavConfPath       	string `json:"navConfPath"`
+	RunningNavVersion 	string `json:"runningNavVersion"`
+	RpcUser 				string `json:"rpcUser"`
+	RpcPassword 			string `json:"rpcPassword"`
 }
 
+func StartConfigManager()  {
+
+	ticker := time.NewTicker(time.Millisecond * 500)
+	go func() {
+		for range ticker.C {
+			LoadUserConfig()
+		}
+	}()
+}
+
+
 // LoadUserConfig loads the config from a file
-func LoadUserConfig() (*Config, error)  {
+func LoadUserConfig() (error)  {
```

Comment:
Is `error` meant to be `err`? Doesn't look like `error` is used at all

Reply:
error is the one of the return types :)

Reply:
this is defining the return types, error is the types, and err is the var :)